docs/tutorial/index.html

```diff
@@ -17,6 +17,194 @@
+<p>Testo was designed to support older OCaml versions starting from 4.08 and to be maintained by the community of users. It is being used to test <a href="https://github.com/semgrep/semgrep">Semgrep</a> which has about 5000 OCaml tests, most of which were originally migrated from Alcotest. Check out the <a href="https://github.com/semgrep/testo/issues">known missing features</a> to see if anything critical to you is missing.</p>
+<h2 id="getting-started">Getting started</h2>
+<h3 id="install-the-testo-library">Install the <code>testo</code> library</h3>
+<p>🚧 <del>Install <code>testo</code> with Opam using <code>opam install testo</code></del> For now, we recommend using <code>testo</code> as a git submodule. Dune will pick it up and build it as part of your project like an ordinary library.</p>
+<h3 id="set-up-your-project">Set up your project</h3>
+<p>The folder <code>tests/snapshots/</code> will be used by Testo to store test snapshots to be tracked by your favorite version control system (git, …). Storing other test data under <code>tests/</code> is encouraged as long as you let Testo manage <code>tests/snapshots/</code>.</p>
+<h3 id="write-a-test-executable">Write a test executable</h3>
+<p>The test executable can be placed anywhere in your Dune project. We recommend having only one such program if possible and calling it <code>test</code>. In this tutorial, we’ll put it in the <code>tests/</code> folder.</p>
+<p>We are going to create the following folders and files:</p>
+<ul>
+<li><code>tests/Test.ml</code>: the entry point of the <code>test</code> program</li>
+<li><code>tests/dune</code>: the Dune file that defines how to build the test executable</li>
+<li><code>tests/snapshots/</code>: created and managed by Testo, under version control</li>
+<li><code>test</code>: a symbolic link to <code>_build/default/tests/test.exe</code></li>
+</ul>
+<p>Create the following <code>tests/dune</code> whose job is to build an ordinary executable named <code>test</code>:</p>
+<pre><code>; Build the test executable for our project
+(executable
+ (name test)
+ (modules Test)
+ (libraries
+    testo
+ )
+)</code></pre>
+<p>We recommend running the test program always from the project root so as to reference any files using paths relative to the project root. Create the symbolic link that will allow us to call the test program directly from the project root:</p>
+<pre><code>$ ln -s _build/default/tests/test.exe test</code></pre>
+<p>If you already have a <code>test</code> file or folder, you may pick another name for the Testo program, it doesn’t matter. The examples in this tutorial assume <code>./test</code> calls our Testo-based test program.</p>
+<p>The last part of this setup is to write the OCaml file <code>tests/Test.ml</code>. Let’s use this:</p>
+<pre><code>(*
+   The entry point for the &#39;test&#39; program that runs the suite of OCaml
+   tests for &lt;this project&gt;.
+*)
+
+let test_hello =
+  Testo.create &quot;hello&quot;
```

Comment:
maybe you can use __FUNCTION__
it's also a good practice.

Reply:
or maybe on this example you can keep it simple like this.

Reply:
We should put a note about `__FUNCTION__` somewhere. I guess it would go in the howtos if not here.